Docs/cvode/source/QSS.rst

```diff
@@ -0,0 +1,97 @@
+.. highlight:: rst
+
+.. role:: cpp(code)
+   :language: c++
+
+Analytically reduced chemistry via quasi-steady state (QSS) assumption in `PelePhysics`
+======================================
+
+.. _sec:subsDiffReacts:
+
+The QSS assumption
+-----------------------
```

Comment:
let's fix these things so that the underline lines up with the title. Modern editors should be able to do that automatically for you.